src/deploy.ts

```diff
@@ -9,17 +9,21 @@ import { upload } from './upload.ts'
 const CONTRACT_TEXT_PREFIX = 't|'
 const CONTRACT_MANIFEST_PREFIX = 'm|'
 
-export async function deploy (args: string[]) {
+export async function deploy (args: string[]): Promise<void> {
   const [urlOrDirOrSqliteFile, ...manifests] = args
   if (manifests.length === 0) throw new Error('missing url or manifests!')
   const toUpload = []
   for (const manifestPath of manifests) {
-    const json = JSON.parse(Deno.readTextFileSync(manifestPath))
-    const body = JSON.parse(json.body)
+    interface ContractBody {
+      contract: { file: string }
+      contractSlim?: { file: string }
+    }
```

Comment:
A few questions here:

- Can this definition be moved above and outside of this function?
- Is this really necessary? If so, I'm a bit skeptical of it as it's not really enforced upon parsing. [Something like Zod](https://zod.dev/) would make more sense as that would actually enforce runtime checks if you wanted them